File: P22_u3.java
/*
 * Click nbfs://nbhost/SystemFileSystem/Templates/Licenses/license-default.txt to change this license
 */

package com.mycompany.p22_u3;
import java.util.Scanner;
/**
 *
 * @author LaboratorioLTI2
 */
public class P22_u3 {

    public static void main(String[] args) {
        Scanner sc= new Scanner (System.in);
         int suma=0, numero;
         
         do{
             System.out.println("Ingrese el numero: ");
             numero = sc.nextInt();
             suma=suma+numero;
             
             
             
         }while(numero!=0);
         System.out.println("Su resultado es: "+suma);
    }
}

File: p22_u3.cs
using System;

namespace p22_u3 // Note: actual namespace depends on the project name.
{
    internal class Program
    {
        static void Main(string[] args)
        {
            int suma = 0, numero;

            do
            {
                Console.WriteLine("Ingrese el numero: ");
                numero = Convert.ToInt32(Console.ReadLine());
                suma = suma + numero;



            } while (numero != 0);
            Console.WriteLine("Su resultado es: " + suma);
        }
    }
}
    

    

